portal-kernel/src/com/liferay/portal/kernel/objectentryportletfilerepository/ObjectEntryPortletFileRepository.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.kernel.objectentryportletfilerepository;
+
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.Repository;
+import com.liferay.portal.kernel.service.ServiceContext;
+
+import org.osgi.annotation.versioning.ProviderType;
+
+/**
+ * @author Alicia García
+ */
+@ProviderType
+public interface ObjectEntryPortletFileRepository {
+
+	public Repository addPortletRepository(
+			long groupId, String portletId, ServiceContext serviceContext)
+		throws PortalException;
+
+	public Repository fetchPortletRepository(long groupId, String portletId);
+
```

Comment:
Why do we need this interface in kernel? It may not even be necessary to have an interface at all if all usages are located in the same module. In that case, we should use a simpler util class.